t/ui/27-plugin_obs_rsync_status_details.t

```diff
@@ -50,21 +50,30 @@ my $host = "http://127.0.0.1:$port";
 sub fake_api_server {
     my $mock = Mojolicious->new;
     $mock->mode('test');
-    $mock->routes->get(
-        '/public/build/:proj/_result' => sub {
-            my $c     = shift;
-            my $proj  = $c->stash('proj');
-            my %repos = (
-                'Proj1'       => 'standard',
-                'BatchedProj' => 'containers',
-            );
-            my $repo = $repos{$proj};
-            $repo = 'images' unless $repo;
-            $c->render(
-                status => 200,
-                text => qq{<result project="$proj" repository="$repo" arch="local" code="published" state="published">}
-            );
-        });
+
+    my $sub = sub {
+        my $c       = shift;
+        my $proj    = $c->stash('proj');
+        my $package = $c->stash('package') // '';
+        $c->render(
+            status => 404,
+            test   => 'unknown package'
+        ) if $proj eq 'Proj2' && $package ne 'mypackage';
```

Comment:
This needs to be `return $c->render(...` or the code after it is still executed.